app/population/views/edit/edit.njk

```diff
@@ -0,0 +1,40 @@
+{% extends "form-wizard.njk" %}
+
+{% macro display_field_group(group) %}
```

Comment:
Wonder if this should be camel case instead of snake case as we use camel case elsewhere (for example components).

Reply:
Sure - I followed the `nunjucks` styleguide by default, forgetting we had a different standard in our app :)